test/integration/conftest.py

```diff
@@ -1,14 +1,28 @@
 import decimal
 import logging
 import os
+import ssl
 import subprocess
 import uuid
+from collections import namedtuple
 from inspect import cleandoc
 from pathlib import Path
+from typing import NamedTuple
 
 import pytest
 
 import pyexasol
+from pyexasol import ExaConnection
+
+
+@pytest.fixture(scope="session")
+def db_version():
+    return os.environ.get("EXA_DB_VERSION", "8.31.0")
```

Comment:
Would it be possible to store the default version (currently 8.31.0) in a common file, or use the string defined in noxconfig here?

Reply:
Accessing the default version from the noxconfig is a bit wild, because you need to find it first

Reply:
It is probably, better to ask the DB which version it has, this can be done via the table EXA_METADATA